templates/registration/profile.html

```diff
@@ -1,54 +1,66 @@
 {% extends "base.html" %}
-{% load humanize %}
 {% load staticfiles %}
+{% load humanize %}
 
 {% block extrastyle %}
     <link rel="stylesheet" href="{% static "css/artist/artist_list.css" %}" type="text/css" charset="utf-8" />
 {% endblock %}
 
 {% block content %}
     <h1>Profile</h1>
-    <h2 class="profile-header">Welcome {% firstof user.get_full_name user.get_username %}!</h2>
-    {% if artists %}
-        <div class="investments">
-            <div class="break">
-                <hr>
-            </div>
-            <h2>Portfolio</h2>
-            <p>Here are the artists that you have invested in:</p>
-            <ul>
-                <div class="row small-up-1 medium-up-2 large-up-4 artistlist">
+    <h2 class="profile-header">Good morning {% firstof user.get_full_name user.get_username %}!</h2>
```

Comment:
Looks like from the merge this was changed back from "Welcome" to "Good morning". Did you want to put that back?